deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4630,6 +4615,22 @@ NAN_METHOD(WebGLRenderingContext::GetParameter) {
       info.GetReturnValue().Set(JS_BOOL(gl->premultiplyAlpha));
       break;
     }
+    case MAX_CLIENT_WAIT_TIMEOUT_WEBGL:
+    case GL_MAX_SERVER_WAIT_TIMEOUT:
+    {
+      info.GetReturnValue().Set(JS_INT(0));
+      break;
+    }
+    case GL_MAX_ELEMENT_INDEX:
```

Comment:
`gl.getParameter(gl.MAX_ELEMENT_INDEX)` was returning -1 but also causing a gl error. I'm not sure why.

Chrome gives `0xffffffff` i.e. `(GLuint)-1`, and this code seems to match Chrome now.

(Why does Chrome work fine?)